src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -167,6 +168,42 @@ class ImageBaseViewer extends BaseViewer {
         this.bindControlListeners();
     }
 
+    /**
+     * Sets the original image width and height on the img element. Can be removed when
+     * naturalHeight and naturalWidth attributes work correctly in IE 11.
+     *
+     * @private
+     * @param {Image} imageEl - The image to set the original size attributes on
+     * @return {Promise} A promise that is resolved if the original image dimensions were set.
+     */
+    setOriginalImageSize(imageEl) {
+        const image = imageEl;
+        const promise = new Promise((resolve, reject) => {
+            // Do not bother loading a new image when the natural size attributes exist
+            if (imageEl.naturalWidth > 0 && imageEl.naturalHeight > 0) {
+                image.originalWidth = imageEl.naturalWidth;
+                image.originalHeight = imageEl.naturalHeight;
+                resolve();
+            } else {
+                const originalImg = new Image();
+                image.originalWidth = 1;
+                image.originalHeight = 1;
+
+                originalImg.error = () => {
+                    reject();
+                };
+                originalImg.onload = () => {
+                    image.originalWidth = originalImg.width || 1;
+                    image.originalHeight = originalImg.height || 1;
+                    resolve();
+                };
+                originalImg.src = imageEl.src;
```

Comment:
I had previously tested out just checking image.width and image.height in the onload function. It appears to have already been styled at that point, hence the need for this. I can go back through and see if there is a point in which the original loaded image can set its data attributes.